pages/windows/cd.md

```diff
@@ -7,6 +7,10 @@
 
 `cd`
 
+- Go to root of current drive:
+
+`cd\`
```

Comment:
I think a space here is more clear
```suggestion
`cd \`
```

Reply:
We can't edit this PR. I will open a new one with another command addition.